test/integration/test_dask.py

```diff
@@ -58,6 +58,11 @@ def test_taskgraph_futures_with_dask(
 
 
 @pytest.mark.skipif(not torch, reason='Torch not installed')
+@pytest.mark.parametrize(
+    'local_dask_client, test_db',
+    [('test_insert_with_dask', 'test_insert_with_dask')],
```

Comment:
(test_insert_with_distributed, test_insert_with_non_distributed)

Reply:
Should we need to use the same database name here?